src/components/Node/Node.wrapper.tsx

```diff
@@ -82,6 +82,9 @@ export const NodeWrapper = ({
   onLinkComplete,
   onLinkCancel,
 }: INodeWrapperProps) => {
+  if (typeof NodeInner === 'undefined') throw new Error('NodeInner is not defined')
+  if (typeof Ports === 'undefined') throw new Error('Ports is not defined')
```

Comment:
Ik weet niet of de check nodig is als je typing goed staat, maar als het dan nog steeds moet blijven zou ik dit doen:
```suggestion
  if (!NodeInner) throw new Error('NodeInner is not defined')
  if (!Ports) throw new Error('Ports is not defined')
```